src/weiroll/index.test.ts

```diff
@@ -0,0 +1,66 @@
+import { CommandFlags, createWeirollDelegateCall } from './index'
+import { Planner, Contract as WeirollContract } from '@weiroll/weiroll.js'
+import { ethers } from 'ethers'
+import { EvmCall } from '../common'
+
+const ERC20_ABI = [
+  'function balanceOf(address account) external view returns (uint256)',
+  'function transfer(address to, uint256 amount) external returns (bool)',
+] as const
+
+// Create DAI contract
+const contract = new ethers.Contract('0x6b175474e89094c44da98b954eedeac495271d0f', ERC20_ABI)
+const daiContract = WeirollContract.createContract(contract, CommandFlags.CALL)
+
+describe('createWeirollTx', () => {
+  it('should handle an empty plan', () => {
+    // GIVEN: a function that doesn't add anything to the planner
+    const addToPlanner = jest.fn()
+
+    const tx = createWeirollDelegateCall(addToPlanner)
+
+    // THEN: the planner was called oince
+    expect(addToPlanner).toHaveBeenCalledTimes(1)
+
+    // THEN: the transaction should be encoded correctly
+    assertDelegateCall(tx, {
+      to: '0x9585c3062Df1C247d5E373Cfca9167F7dC2b5963',
+      data: '0xde792d5f0000000000000000000000000000000000000000000000000000000000000040000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000',
```

Comment:
Is there any way to verify this calldata?
Yes, the test is asserting that's what's returned, but other than running the test to get the output then pasting it here, how do we get to this calldata for this weiroll call?